aws/sdk/examples/lambda/src/bin/invoke-function.rs

```diff
@@ -137,7 +137,9 @@ async fn main() {
         //
         // For our example, we will simply print that the function doesn't
         // exist and return a non-zero exit code to indicate the failure.
-        Err(SdkError::ServiceError { err, .. }) if err.is_resource_not_found_exception() => {
+        Err(SdkError::ServiceError { err, .. })
```

Comment:
you're on an old SDK, you need to update

Reply:
you can just back this out entirely, the code that was there before is the pattern we want to guide people towards